### PR TITLE
chore: fixed video

### PR DESCRIPTION
**Pull request by jpvajda**
This PR fixes a broken video and also linter fixes.

### CODE REVIEW

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx [hunk 1]
```diff
@@ -627,19 +629,29 @@ If you need assistance with New Relic Products, you are in good hands with sever
...
-  </Collapser>
-
-  <Collapser
-    id="nrdiag"
-    title="Diagnostic tools"
-  >
-    New Relic offers a [diagnostics utility](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics) that can automatically detect common problems with New Relic agents. If Diagnostics detects a problem, it suggests troubleshooting steps. New Relic Diagnostics can also automatically attach troubleshooting data to a New Relic Support ticket. We have also made available [Troubleshooting Frameworks](https://discuss.newrelic.com/t/troubleshooting-frameworks/108787) that step users through common troubleshooting questions.
-  </Collapser>
+{' '}
+<Collapser id="github" title="Github">
```
Comment (jerelmiller) [inline src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:633]: It looks like prettier might have mangled this file a bit. Would you be willing to undo these changes and redo the video change without formatting this file?

**Review by jerelmiller — CHANGES_REQUESTED** [review]

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx (hunk 1 above)
Comment (jpvajda) [reply src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:633]: Sure.. curious as to why, it was a very simple change and I just saved the file.
